packages/ui/components/form/checkbox/Checkbox.tsx

```diff
@@ -56,11 +56,12 @@ const CheckboxField = forwardRef<HTMLInputElement, Props>(
                     disabled={disabled}
                     id={rest.id ? rest.id : id}
                     className={classNames(
-                      "text-primary-600 focus:ring-primary-500 border-default bg-default h-4 w-4 rounded ltr:mr-2 rtl:ml-2",
+                      "text-primary-600 focus:ring-primary-500 border-default bg-default focus:bg-default active:bg-default h-4 w-4 rounded checked:hover:bg-gray-600 focus:outline-none focus:ring-0 ltr:mr-2 rtl:ml-2",
```

Comment:
I know this wasnt in the design file i sent you 

But i wonder if it makes sense for us to use the users brand colour here. We do the same for buttons and switches so i think it could fit in quiet nicely. 

I'm happy to pick this up if you dont have time <3

Reply:
ok sure i will make these changes. Currently on road.